### ISSUE TITLE
'NoneType' object is not subscriptable

### ISSUE THREAD

**Issue by pejotigrek** [n4ts/ha-notione#1]
not sure if this is the newest version issue, or was here earlier as I had the component turned off for some time, but after update I can see this in HomeAssistant's log:

```
2021-02-07 23:55:30 ERROR (MainThread) [homeassistant.components.device_tracker] Error setting up platform legacy
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/device_tracker/legacy.py", line 239, in async_setup_legacy
    setup = await hass.async_add_executor_job(
  File "/usr/local/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/notione/device_tracker.py", line 43, in setup_scanner
    NotiOneTracker(hass, config, see)
  File "/config/custom_components/notione/device_tracker.py", line 68, in __init__
    update_interval(None)
  File "/config/custom_components/notione/device_tracker.py", line 63, in update_interval
    self._update_info()
  File "/config/custom_components/notione/device_tracker.py", line 111, in _update_info
    battery = dev['notiOneDetails']['battery']
TypeError: 'NoneType' object is not subscriptable
```

**Comment by n4ts** [comment]
@pejotigrek this the same case like in other issue. Can you send me output from https://api.notinote.me/secured/internal/devicelist after login?

**Comment by pejotigrek** [comment]
understood. so [here's the output](https://github.com/n4ts/ha-notione/issues/2#issuecomment-784007731) - it gave me a 403 error.

**Comment by pejotigrek** [comment]
I guess the project died?

**Comment by n4ts** [comment]
fixed [bc0357b](https://github.com/n4ts/ha-notione/commit/bc0357b08f02bff393a96dc12ea8e7979e550ffa)

**Comment by pejotigrek** [comment]
thanks. log is clean, issue seems to be fixed! :)
